Library/Formula/open-mpi.rb

```diff
@@ -41,6 +41,7 @@ def install
       --enable-ipv6
       --with-libevent=#{Formula["libevent"].opt_prefix}
       --with-sge
+      --with-platform=optimized if build.head?
```

Comment:
This is adding `--with-platform-optimized`, `if`, and `build.head?` as three separate arguments.  Instead add this below:
```ruby
args << `--with-platform-optimized` if build.head?